app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, first_file, second_file = command.split()
+    second_file_part = os.path.split(second_file)
+    directories_list = second_file_part[0].split("/")
```

Comment:
do you need this `directories_list = second_file_part[0].split("/")` ? Can you create directories without it?)

Reply:
U r right. I don't need several lines. Thanks